charts/beta/vector/values.yaml

```diff
@@ -71,6 +84,10 @@ vector:
         items:
           - key: ca.pem
             path: ca.pem
+    - name: vector-aws
+      configMap:
+        defaultMode: 420
+        name: vector-cloudtrail-aws-config
```

Comment:
should this be marked optional? so that we only mount it when cloudtrail is configured?